packages/world/src/modules/core/implementations/AccessManagementSystem.sol

```diff
@@ -35,4 +36,20 @@ contract AccessManagementSystem is System {
     // Revoke access from the given resource
     ResourceAccess.deleteRecord(resourceSelector, grantee);
   }
+
+  /**
+   * Transfer ownership of the given namespace to newOwner.
+   * Revoke ResourceAccess for previous owner and grant to newOwner.
+   * Requires the caller to own the namespace.
+   */
+  function transferOwner(bytes16 namespace, address newOwner) public virtual {
```

Comment:
We should probably name this `transferOwnership`, since the ownership, not the owner, is the thing being transferred, and also aligns with e.g. [OpenZeppelin's Ownable](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol).

Reply:
Yeah, that makes more sense - thanks!